element_deeplabcut/readers/dlc_reader.py

```diff
@@ -383,3 +383,67 @@ def do_pose_estimation(
         allow_growth=allow_growth,
         use_shelve=use_shelve,
     )
+
+
+def do_pytorch_pose_estimation(
```

Comment:
Do we need this wrapper function anymore?

Reply:
We don't. I added this just a few hours before we merged your changes to `main`, making this function unnecessary. I'll remove it in the next commit